server/src/main/java/io/spine/server/entity/EntityLifecycle.java

```diff
@@ -264,6 +269,26 @@ public final void onStateChanged(EntityRecordChange change,
         postIfRestored(change, dispatchedMessageIds);
     }
 
+    public final void onInvalidEntity(MessageQualifier cause,
```

Comment:
Please document.

Also `cause` parameter name seems confusing, as we also have `error` right here. They kinda interfere.